qiskit/primitives/containers/sampler_pub_result.py

```diff
@@ -25,7 +25,13 @@
 
 
 class SamplerPubResult(PubResult):
-    """Result of Sampler Pub."""
+    """Result of Sampler Pub.
+
+    This is a :class:`.SamplerV2`-specific subclass of :class:`.PubResult` that adds helper methods
+    to deal with the bit-array like :class:`.BitArray` data directly (implicitly going via the
+    :class:`.DataBin` in :attr:`data`).  See :class:`.PubResult` for more documentation on how to
+    use this class.
```

Comment:
Double checking: `)` and `-` can immediately follow a backtick in ReST without escapes?